cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -553,3 +554,7 @@ plutusV3SupplementalDatumScript =
 
 
```

Comment:
Similarly to `plutusV3SupplementalDatumScript` can you add a haddock as to how this was created?

Reply:
That's problematic, because it's a script from https://github.com/IntersectMBO/cardano-cli/issues/1023 , and @mkoura received it from @catch-21 , so we don't know how it was generated.

It's the same stake-script as in `scripts/plutus/scripts/v2/stake-script.plutus`.

Reply:
I see a [comment](https://github.com/IntersectMBO/cardano-node/blob/17b20815064f2a018be3b33614d927705fdba5d1/cardano-testnet/src/Testnet/Defaults.hs#L550) that suggests you'll find the definition here: https://github.com/IntersectMBO/cardano-node/blob/master/bench/plutus-scripts-bench/src/Cardano/Benchmarking/PlutusScripts/SupplementalDatum.hs

Reply:
@catch-21 thanks for jumping in! No I meant plutusV2StakeScript which I've added, and copied from the attachment in https://github.com/IntersectMBO/cardano-cli/issues/1023

Reply:
I don't recall the definition of this one. It was introduced in https://github.com/IntersectMBO/cardano-node-tests/pull/1353 so maybe @saratomaz can find out from an old conversation. Note, this script is identical to their [anyone-can-mint.plutus](https://github.com/IntersectMBO/cardano-node-tests/blob/master/cardano_node_tests/tests/data/plutus/v2/anyone-can-mint.plutus).